exampleCourse/infoCourse.json

```diff
@@ -3,7 +3,8 @@
     "name": "XC 101",
     "title": "Example Course",
     "options": {
-        "useNewQuestionRenderer": true
+        "useNewQuestionRenderer": true,
+        "isExampleCourse": true
```

Comment:
I'm worried that people are going to copy this. What about an evil hack that directly detects the example course? For example, when loading the course we could check the `uuid`, `name`, and `title` and set `isExampleCourse` if they all match. Then we won't need an option in the JSON file and no one will accidentally copy it. There is a slight risk if someone copies the entire example course, including the UUID, but that will break anyway sooner or later when we have a UUID conflict with the real example course.

Reply:
I agree there's a risk the file would get copied. It just seemed cleaner, more controllable with a variable than trying to make decisions based on other contents. I'll work on it and see what we can do.

Reply:
I agree that detecting exampleCourse directly is a hack, but it's only this one case that we'll ever have to worry about, so I don't mind so much. I really want to avoid confusion with people copying things from the example course.

Reply:
Do you have a preference about doing the dirty hack at some higher level (`lib-coursedb`) or putting it in `instructorQuestion` where it's used?

Reply:
I have a slight preference for putting it at a higher level. I suspect that @tbretl's editor patches might also want to handle the example course specially?